packages/racer/src/controllers/results.ts

```diff
@@ -61,7 +61,7 @@ const selectResponseTypeByHeader = (
       MIME_HTML
     );
   } else {
-    return new EndpointResponse(record.results.lhr).withHeader(
+    return new EndpointResponse(record.results.steps).withHeader(
```

Comment:
did you end up re-writing the normal `profile` command to return a step of 1 here?

Reply:
Yes both commands have been tested to work